### ISSUE TITLE
Cleanup readme

### ISSUE THREAD

**Issue by Lex-2008** [Lex-2008/backup3#42]
first mention features/benefits:
* intellectual cleanup - main differentiator (I haven't seen it anywhere else)
  * starts deleting old files only when you're running out of disk space
  * prioritises hourly, daily, weekly, monthly backups ("daily" backup is state of backup dir at midnight, "monthly" - at midnight of first day of the month, etc) - so you have same _number_ of daily backups as monthly
    * actually based not on _number_ of backups, but their _age_ - so before deleting a backup which was made 4 months ago, it will delete a backup which was made 5 days ago
* all features of rsync (since it's most often used for file transfer)
  * incremental backup - transfers only changed files, keeping history of several versions
  * remote backup
    * from multiple _remote systems_ to single _backup server_
    * minor requirements to _remote systems_ - directory with valuable data should be made available to _backup server_ via any of the following means:
      * either via rsync server which is configured and running
      * or as shared folder
      * or via SSH access
      * exotic methods are also possible (like using external hard drive to backup air-gapped _remote systems_)
    * "pull" remote sync - if a _remote system_ is compromised - it can't wipe backups (it can fill your backups with garbage, however - but that's true for any backup system)
    * read-only access from _backup server_ to _remote systems_ - if _backup server_ is compromised - it can't wipe other systems (it can wipe your backups, however - but that's true for any backup system which doesn't use write-once medium like CD-R)
  * transparent protection against network transfer errors (when backing up from rsync server) - if a file is damaged due to network fluke, it will be retransferred
  * filtering of files to backup based on their path/name/ext
* lack of vendor lock-in - not using proprietary file format
  * all files in backup are stored as plain files
  * you can access them using any standard file manager
  * it will be more correct after https://github.com/Lex-2008/backup3/issues/36 is implemented - since currently files are renamed
* easy to navigate WebUI
  * restore any file or whole dir
  * password-protect any dir
  * remotely trigger backups
* parity files to protect against bit rot
* scheduled backup (scheduled manually via cron)
* encryption (configured manually via encfs)

then installation on all platforms
* debian/ubuntu, centos/rhel
* basically, prepend `$package_manager install` to first two lines of https://github.com/Lex-2008/backup3#requirements

then configuration (simple/advanced usage), WebUI, and all other complications can be moved to a separate "extra features" file
